website/README.md

```diff
@@ -63,6 +63,14 @@ If you're doing active development we suggest the following workflow:
     navigate to `http://localhost:1080`. Check the email listed and click the
     log in link. You're now logged in and authenticated.
 
+### Using debug user credentials
+
+If you want to quickly test the website, you can use the debug user credentials workflow:
```

Comment:
To simplify development, I think its reasonable to always activate this.  What about putting it into the checked in `.env` file?